meta/documents/user_guide/en/updating.md

```diff
@@ -6,37 +6,61 @@ The Wayfair plugin may be updated at any time to ensure that it is providing the
+
     * **Do not select the deprecated, similarly named `Plugin overview` option, if it appears in the menu.**
 
     * The screen should show a list of plugin sets, with a set of buttons for each one.
 
 2. Locate the Plugin Set where the Wayfair plugin is installed.
 
+     ![linked clients](../../../images/en/plugin_sets_linked_clients.png)
+
     * The list of plugins for the Plugin Set are hidden behind the `Edit plugin set` button.
 
-3. Click the `Edit plugin set` button for the Plugin Set.
+3. Click on the `Edit` button ![pencil button](../../../images/common/button_pencil.png) button for the Plugin Set.
+
+4. Verify that a row exists on the page with the `Name` displayed as "Wayfair" and that the `Deployed` column is also populated, indicating that the plugin is in use:
+
+    ![wayfair plugin needs update](../../../images/en/installation/wayfair_plugin_needs_update.png)
+
+5. Hover over the `Update` button ![update button](../../../images/common/button_update.png) in the `Actions` column.
+
+5. If the button is disabled, you already have the latest version of the Wayfair plugin. **Do not continue with this guide**:
 
-4. Verify that a row exists on the page with the `Name` displayed as "Wayfair" and that the `Deployed` column is also populated, indicating that the plugin is in use.
+    ![no update](../../../images/en/installation/no_update.png)
 
-5. Hover over the `Update` button in the `Actions` column
 
-5. If the button displays a `No update available` message, you already have the latest version of the Wayfair plugin. **Do not continue with this guide.** Otherwise, it should show `Update available` and a version number.
+6. Click the `Update` button ![update button](../../../images/common/button_update.png). A page will be presented that has the Wayfair logo, information about the Wayfair plugin, and a new, different `Update` button at the top:
 
-6. Click the `Update` button. A page will be presented that has the Wayfair logo, information about the Wayfair plugin, and a new, different `Update` button at the top.
+    ![update page](../../../images/en/installation/update_page.png)
 
 7. Review the information on the page.
 
 9. In the `Select version` drop-down menu, choose [the newest plugin version](https://github.com/wayfair-contribs/plentymarkets-plugin/releases) if it is not already selected.
 
-10. Click the `Update` button at the top of the page. After a short pause, you will be redirected to the individual Plugin Set's details.
+    ![update selector](../../../images/en/installation/update_version_selection.png)
 
-11. Verify that a row exists on the page with the `Name` displayed as "Wayfair" and the `Installed` version showing as the desired version
+10. Click the `Update` button ![update button](../../../images/en/updating/update_button.png) at the top of the page. After a short pause, you will be redirected to the individual Plugin Set's details.

```

Comment:
The image has a broken link when I was looking at the markdown.

Reply:
thanks. yeah I forgot to refactor that link. There's no `updating` image directory anymore. It's all in `installation` now.